devops/arangodb.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - hosts: all
+  become: yes
```

Comment:
This should not go here. You can add `become: yes` to any individual task that requires root access instead for a safer provisioning setup.

Reply:
Fixed in cfed808